maintainers/scripts/format-maintainer-list/src/main.rs

```diff
@@ -0,0 +1,153 @@
+#![warn(clippy::pedantic)]
+
+use serde::Deserialize;
+use std::{
+    collections::HashMap,
+    env,
+    fmt::Debug,
+    process::{Command, ExitCode},
+};
+
+#[derive(Debug, Deserialize)]
+struct Maintainer {
+    name: String,
+    email: String,
+    matrix: Option<String>,
+    github: Option<String>,
+    #[serde(rename = "githubId")]
+    github_id: Option<u64>,
+    keys: Option<Vec<Key>>,
+}
+
+#[derive(Debug, Deserialize)]
+struct Key {
+    fingerprint: String,
+}
+
+fn print_key_value(key: &str, value: impl Debug) {
+    println!("    {key} = {value:?};");
+}
+
+fn print_optional_key_value(key: &str, value: Option<impl Debug>) {
+    if let Some(v) = value {
+        print_key_value(key, v);
+    }
+}
+
+const PRELUDE: &str = r#"/* List of NixOS maintainers.
+    ```nix
```

Comment:
do we have support for markdown syntax in comments?

Reply:
This is taken from the existing file.

Reply:
my bad, I think i looked at the wrong file the first time around